src/MCPClient.Dockerfile

```diff
@@ -1,97 +1,11 @@
-FROM ubuntu:14.04
+FROM artefactual/archivematica-mcp-client-base:20180219.01.52dc9959
```

Comment:
We could have a custom image for JiscRDSS fork but at this point they share the same dependencies.